tests/e2e/e2e_suite_test.go

```diff
@@ -9,76 +9,59 @@ import (
 	"testing"
 
 	"github.com/hashicorp/go-uuid"
+	"github.com/matt-royal/biloba"
+	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/sclevine/spec"
 	"k8s.io/client-go/kubernetes/scheme"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-
 	controllerruntime "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 	hncv1alpha2 "sigs.k8s.io/hierarchical-namespaces/api/v1alpha2"
 )
 
 var (
-	suite             spec.Suite
 	testServerAddress string
-	g                 *WithT
 	k8sClient         client.Client
 	rootNamespace     string
 	apiServerRoot     string
 )
 
-func Suite() spec.Suite {
-	if suite == nil {
-		suite = spec.New("E2E Tests")
-	}
-
-	return suite
+func TestE2E(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecsWithDefaultAndCustomReporters(t, "E2E Suite", biloba.GoLandReporter())
 }
 
-func SuiteDescribe(desc string, f func(t *testing.T, when spec.G, it spec.S)) bool {
-	return Suite()(desc, f)
-}
-
-func TestSuite(t *testing.T) {
-	g = NewWithT(t)
-
-	beforeSuite()
-	defer afterSuite()
-
-	suite.Run(t)
-}
-
-func beforeSuite() {
+var _ = BeforeSuite(func() {
 	apiServerRoot = mustHaveEnv("API_SERVER_ROOT")
 
 	logf.SetLogger(zap.New(zap.WriteTo(os.Stderr), zap.UseDevMode(true)))
 
 	hncv1alpha2.AddToScheme(scheme.Scheme)
 
 	config, err := controllerruntime.GetConfig()
-	g.Expect(err).NotTo(HaveOccurred())
+	Expect(err).NotTo(HaveOccurred())
 
 	k8sClient, err = client.New(config, client.Options{Scheme: scheme.Scheme})
-	g.Expect(err).NotTo(HaveOccurred())
+	Expect(err).NotTo(HaveOccurred())
 
 	rootNamespace = mustHaveEnv("ROOT_NAMESPACE")
 	ensureServerIsUp()
-}
+})
 
-func afterSuite() {
-}
+var _ = AfterSuite(func() {
```

Comment:
If we don't have any AfterSuite logic, we should just delete this.

Reply:
Right. I have removed it.